File: WebApp/Frond-end-for-admin/src/app/Admin/Model/OrderItems.ts
import { Orders } from './Orders';


export class OrderItems{

    // private long id;
    // private long item_id;
    // private int quantity;
    // private double price;
    // private Orders orders;



    constructor(
        public id: number,
        public item_id: number,
        public quantity: number,
        public price: number,
        public orders: Orders
    ) { }

    get getId() {
        return this.id;
    }

    set setId(id: number) {
        this.id = id;
    }


  

    get getItemId() {
        return this.item_id;
    }

    set setItemId(item_id: number) {
        this.item_id = item_id;
    }

    get getQuantity() {
        return this.quantity;
    }

    set setQuantity(quantity: number) {
        this.quantity = quantity;
    }

    get getPrice() {
        return this.price;
    }

    set setPrice(price: number) {
        this.price = price;
    }

    get getOrders() {
        return this.orders;
    }

    set setOrders(orders: Orders) {
        this.orders = orders;
    }

    public toString() {
        return "Orders [id=" + this.id + ", item_id=" + this.item_id + ", quantity=" + this.quantity + ", price=" + this.price + ", orders=" + this.orders + "]";
    }

}

File: ArApp3/ArApp3/app/src/main/java/com/example/arapp3/Model/OrderItems.java
package com.example.arapp3.Model;

public class OrderItems {
    private long id;
    private long item_id;
    private int quantity;
    private double price;

    public OrderItems(){}

    public OrderItems(long item_id, int quantity, double price) {
        this.item_id = item_id;
        this.quantity = quantity;
        this.price = price;
    }

    public OrderItems(long id, long item_id, int quantity, double price) {
        this.id = id;
        this.item_id = item_id;
        this.quantity = quantity;
        this.price = price;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getItem_id() {
        return item_id;
    }

    public void setItem_id(long item_id) {
        this.item_id = item_id;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }
}
